GameServer/Logic/Content/ItemManager.cs

```diff
@@ -8,18 +8,18 @@
 using System.Threading.Tasks;
 using LeagueSandbox.GameServer.Logic.Items;
 using LeagueSandbox.GameServer.Core.Logic.PacketHandlers;
+using Ninject;
 
 namespace LeagueSandbox.GameServer.Logic.Content
 {
     public class ItemManager
     {
-        private Game _owner;
         private Dictionary<int, ItemType> _itemTypes;
 
-        private ItemManager(Game owner)
+        public ItemManager()
         {
-            _owner = owner;
             _itemTypes = new Dictionary<int, ItemType>();
+            LoadItems();
```

Comment:
I don't think constructors should cause any heavy procedures to run, thus why this was implemented differently originally. Up to debate though, would like to hear your opinion on it.